### PR TITLE
[15.0][ADD] account_invoice_clearing

### PR DESCRIPTION
**Pull request by Shide**
New module to allow invoice clearing between invoices.

With this wizard, you can clear **out invoices** with **in invoices** and _vice versa_.

MT-2035 @moduon @rafaelbn @yajo please review

### CODE REVIEW

File: account_invoice_clearing/static/description/icon.svg
Comment (yajo) [inline account_invoice_clearing/static/description/icon.svg:1]: Please remove the SVG. It's not needed when it's the standard one.

File: account_invoice_clearing/tests/test_clearing_wizard.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                    (0, 0, {"name": "Clearing Contact 1", "type": "invoice"}),
+                    (0, 0, {"name": "Clearing Contact 2", "type": "invoice"}),
+                ],
+            }
+        )
+        cls.sorted_by_seq = lambda l: sorted(l, key=lambda x: x.sequence)
+
+    def test_clearing_wizard(self):
+        def sorted_by_sequence(lines):
+            return sorted(lines, key=lambda l: l.sequence)
```
Comment (yajo) [inline account_invoice_clearing/tests/test_clearing_wizard.py]: It seems to me like both `sorted_by_seq` and `sorted_by_sequence` are the same thing. Can't we drop one?

**Review by yajo — APPROVED** [review]
Code review, just 2 minor comments. Really excellent code! 👏🏼👏🏼👏🏼 🏆 ✨

File: account_invoice_clearing/tests/test_clearing_wizard.py (hunk 1 above)
Comment (Shide) [reply account_invoice_clearing/tests/test_clearing_wizard.py]: Yes, I forgot to delete one 😸

**Review by rafaelbn — APPROVED** [review]
Perfect! Tested in production, staing and runboat! 👍🏼

**Pull request merged**